server/src/utils/chess-game-util.js

```diff
@@ -0,0 +1,56 @@
+const calcExpectedScore = (_ratingA, _ratingB) => {
+  const ratingA = parseInt(_ratingA, 10);
+  const ratingB = parseInt(_ratingB, 10);
+  const score = 1 / (1 + (10 ** ((ratingB - ratingA) / 400)));
+  return parseFloat(score.toFixed(2));
+};
+
+const calcScores = (winner, pOneExpScore, pTwoExpScore) => {
+  let pOneScore;
+  let pTwoScore;
+
+  switch (winner) {
+    case 'player_one':
```

Comment:
This is a hard string constrain b/w client & server. This seems to be an ENUM. Which we don't have the luxury of. Possible solutions could be null-boolean (as we have three states), or a config file (which is not great) as both of them need to keep it anyway. Or may be a frozen object, which may be little better.

Reply:
This reminds me we also need to freeze our configuration files everywhere.